README.md

```diff
@@ -107,3 +107,32 @@ For more information see the [Code of Conduct FAQ](https://opensource.microsoft.
 contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
 
 For more details, please see [CONTRIBUTING.md](./CONTRIBUTING.md), or the [contribution guide](https://docs.microsoft.com/quantum/contributing/).
+
+## [Optional] Using Prerelease Versions ##

```

Comment:
In hindsight, I'm wondering if this adds more noise than being helpful.
Reading it, it appears people have to do something to enable the prerelease feed and if they don't they will be using released bytes, when the matter of fact is that they are already using it if building from master. 

Maybe instead of instructions, we just need to give notice that when building from `master` you will be using pre-release packages picked up from our public pre-release feed configured on the nuget.config in the repo?

Reply:
That's not even true - the config only makes it possible to reference packages from the prerelease feed, but looking at the PR, I think all packages still point to the released ones. So what it really should say is that the prerelease feed is added for all code in this repo and you can use prerelease packages if you want to.